### ISSUE TITLE
[Suggestion] Respect no_default_monsters tag

### ISSUE THREAD

**Issue by emihead** [Peeko32213/UnusualFish2#11]
Hi! I'm here because I'm getting a ton of Prawn spawning in biomes that use Forge's no_default_monsters tag. Since vanilla monsters respect that tag, the entire mob cap goes to Prawn. Modpack authors can fix this issue with other mods, but it'd be really helpful if this mod respected that tag by default.

**Comment by Coda1552** [comment]
Hi, I've encountered this too and I thought I had fixed it in the latest version, perhaps I haven't...what version are you using?

**Comment by emihead** [comment]
I'm on 1.0.7.
If you're using is_cave or is_overworld, that might be the issue. The original mod doesn't use those tags, I believe in an attempt to brute-force remove other mod spawns, but they're good for compat so I use a sidemod that adds them to the cave biomes. forge:no_default_monsters is what AC and other mods like Terralith are using to tell other mods to not spawn anything in certain biomes, so it would be good for compat if you aren't using it already. If you are, though, then I'm honestly not sure what's going on.

**Comment by Coda1552** [comment]
Is forge:no_default_monsters a biome tag?

**Comment by emihead** [comment]
Yep. [here](https://github.com/AlexModGuy/AlexsCaves/blob/main/src/main/resources/data/forge/tags/worldgen/biome/no_default_monsters.json) is AC adding to it. To my understanding, Forge doesn't add it by default, it's just defined manually by mods that use it.

**Comment by Coda1552** [comment]
Ah, that explains why I couldn't find it. The mod is going through a big QoL update right now, so I'll include this fix in there. Thanks for reporting :)
